platform/asterix/boot/src/drivers/watchdog.c

```diff
@@ -7,10 +7,15 @@ void watchdog_init(void) {
 	NRF_WDT->RREN = WDT_RREN_RR0_Enabled << WDT_RREN_RR0_Pos;
 	/* WDT expiration: 8s */
 	NRF_WDT->CRV = 32768 * 8;
+	NRF_WDT->TASKS_START = 1;
+	// NOTE: at this point WDT can no longer be stopped, it will even survive
+	// a system reset!
```

Comment:
Yeah, I agree with this change, but man that's an annoying "feature" of nRF52.  Amusingly, it will *not* survive a watchdog reset!  So one option is to wait for the watchdog to bark.  But at an 8 second interval, you could be waiting a long time.  Another option, I guess, would be to go to system-off and ask nPM to reset us.  But man those are all bad options.  I think I agree that this is the best option.

Reply:
I thought about forcing WDT reset, but 8s feels a bit too much for a system reset (e.g. when triggering an update). Not sure how such "feature" got in TBH.